spec/manifest_pr_checker_spec.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require_relative 'spec_helper'
+
+module Danger
+  describe Danger::ManifestPRChecker do
+    it 'should be a plugin' do
+      expect(Danger::ManifestPRChecker.new(nil)).to be_a Danger::Plugin
+    end
```

Comment:
Once the three plugins (this plus #1 and #2) are all on `trunk`, this check could be made into a shared example or utility method using [`described_class`](https://apidock.com/rspec/Spec/Example/ExampleGroupMethods/described_class).

I haven't tried it, but I think this should work

```suggestion
    it 'should be a plugin' do
      expect(described_class.new(nil)).to be_a Danger::Plugin
    end
```